### ISSUE TITLE
elm analyze doesn't refresh

### ISSUE THREAD

**Issue by xDelph** [elm-tooling/elm-language-client-vscode#64]
Just install the plugin, I have elm, elm-test and elm-format installed correctly

but when I fix for example an unused import, it don't get ride of the message.
also, when I removed an usefull import, I haven't the error appearing anymore that it don't find the type/variable/function definition

## Expected Behavior

quick fix remove the useless import, parenthesis, display the error (like missing function/type definition/import

## Current Behavior

the message keep appears when we remove an useless import

## Steps to Reproduce (for bugs)

- just write some useless import and some good one
- remove both useless and good one
- nothing change in the view (no warning removed, no new error)

## Context

previous elm plugin was great to see elm error within vscode, which don't work anymore now

## Your Environment

* Version used: elmtooling.elm-ls-vscode 0.7.4
* Editor name and version (e.g. VSCode 1.36.1): VSCode  1.41.1
* Environment name and version (e.g. node.js 5.4): node v10.15.3
* Operating System and version: Mac OS catalina and Ubuntu 18.10

**Comment by Razzeee** [comment]
Removing an obsolete import removes the warning for me.

If you remove a used import, you will need to save your file, as we rely on the compiler to validate at them moment. And there is no way to feed the elm compiler in memory. After saving you should get an error.

Did you test this with the spa-example? https://github.com/rtfeldman/elm-spa-example

**Comment by xDelph** [comment]
Hello, just test with elm-spa example, the warning disappear (my own project should be too big or something)
but for the errors, they don't appear when i remove a good import (like the Profile one in Author.elm)

**Comment by Razzeee** [comment]
Right, I thought you were suggesting that the underlining does not disappear in the code view. I can confirm, that the message doesn't seem to disappear from the problems list.

If I remove Profile and save the file I'm getting the error.

**Comment by xDelph** [comment]
so at least we agree that there is a problem with the warning ?

on side the error don't appear, even on clean branch
here is my vscode setting, perhaps something wrong here :

`{
  "git.autofetch": true,
  "editor.multiCursorModifier": "ctrlCmd",
  "git.confirmSync": false,
  "code-runner.executorMap": {
    "fsharp": "fsharpi"
  },
  "editor.insertSpaces": true,
  "eslint.enable": true,
  "javascript.format.enable": false,
  "FSharp.fsiFilePath": "/usr/bin/fsharpi",
  "csharp.suppressDotnetInstallWarning": true,
  "FSharp.fsacRuntime": "netcore",
  "csharp.suppressDotnetRestoreNotification": true,
  "window.zoomLevel": 0,
  "javascript.updateImportsOnFileMove.enabled": "always",
  "diffEditor.ignoreTrimWhitespace": false,
  "editor.minimap.enabled": false,
  "breadcrumbs.enabled": false,
  "[javascript]": {
    "editor.tabSize": 2,
  },
  "[elm]": {
    "editor.tabSize": 4,
    "editor.formatOnSave": true,
    "editor.defaultFormatter": "SimonSiefke.prettier-vscode",
  },
  "[fsharp]": {
    "editor.tabSize": 4
  },
  "editor.formatOnSave": true,
  "liveshare.featureSet": "insiders",
  "docker.images.groupBy": "Repository",
}`

elm and elm-format work on the terminal should no problem on this side

**Comment by Razzeee** [comment]
Yes, warnings seem buggy in that regard, will look at that whenever I find some time.

I don't think your settings are problematic, did you check out the log tab for the project? Maybe you can find something going wrong there.
At the moment you should be getting an error in your error list, as soon as you safe, as the package website is down.

**Comment by xDelph** [comment]
the log tab ? what do you mean ?
i check the problems and output panel for elm, didn't see any error (about the profile) (just all the warning (167) in the problems panel)

and i get the "TROUBLE VERIFYING DEPENDENCIES - I could not connect to https://package.elm-lang.org to get the latest list of\npackages..."

**Comment by Razzeee** [comment]
You should have a log under `Output` > `Elm (elm-spa-example)`  after opening an elm file.

**Comment by xDelph** [comment]
Received save for file:///home/tdelalonde/projects/perso/elm-spa-example/src/Author.elm
[Info  - 10:29:43 AM] Diagnostics were requested due to a file open or save
[Info  - 10:29:44 AM] A code lens was requested for file:///home/tdelalonde/projects/perso/elm-spa-example/src/Author.elm
[Info  - 10:29:44 AM] A code lens was requested for file:///home/tdelalonde/projects/perso/elm-spa-example/src/Author.elm
[Info  - 10:29:44 AM] A hover was requested
[Info  - 10:29:44 AM] A code lens resolve was requested
[Info  - 10:29:44 AM] A code lens resolve was requested
[Info  - 10:29:44 AM] A code lens resolve was requested
[Info  - 10:29:44 AM] A code lens resolve was requested
[Info  - 10:29:44 AM] A code action was requested


no message about any error or something

**Comment by xDelph** [comment]
just retest the error problem, seems to work for elm-spa-example
but not in my workspace

i think this can be related to #66, as i have multiple elm project in it

**Comment by nahiyan** [comment]
> just retest the error problem, seems to work for elm-spa-example
> but not in my workspace
> 
> i think this can be related to #66, as i have multiple elm project in it

I do think so too. Can you check if the extension works for only one Elm project in the workspace, but not for others? If so, then #66 is pretty much the same issue as this.
